sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosConstants.java

```diff
@@ -18,4 +31,20 @@
      * @see com.mesosphere.sdk.state.DefaultSchemaVersionStore
      */
     public static final String SERVICE_ROOT_PATH_PREFIX = "/dcos-service-";
+
+    public static boolean networkSupportsPortMapping(String networkName) {
+        boolean supportsPortMapping;
+        switch (networkName) {
+            case "dcos":
+                supportsPortMapping = false;
+                break;
+            default:
+                supportsPortMapping = DEFAULT_PORT_MAPPING_POLICY;
+        }
+        return  supportsPortMapping;
+    }
+
+    public static boolean isSupportedNetwork(String networkName) {
```

Comment:
I removed it, then forgot to add it back. It's now used when parsing the YAML and when building the `networkInfo`s.